cmd/cluster-agent/api/server.go

```diff
@@ -123,14 +123,18 @@ func StartServer(ctx context.Context, w workloadmeta.Component, taggerComp tagge
 		return struct{}{}, nil
 	})
 
+	const maxMessageSize = 4 * 1024 * 1024 // 4 MB
+
 	opts := []grpc.ServerOption{
 		grpc.StreamInterceptor(grpc_auth.StreamServerInterceptor(authInterceptor)),
 		grpc.UnaryInterceptor(grpc_auth.UnaryServerInterceptor(authInterceptor)),
+		grpc.MaxSendMsgSize(maxMessageSize),
+		grpc.MaxRecvMsgSize(maxMessageSize),
```

Comment:
the default message size in grpc is 4MB. 

I added these explicitly here in order to make sure the tagger server gets the same message max size when splitting the response into chunks.